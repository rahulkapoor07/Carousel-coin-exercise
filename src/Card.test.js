import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";

test("should render", ()=>{
    render(<Card />);
});

test("should snapshot 1", ()=>{
    const {asFragment} = render(<Card />);
    expect(asFragment()).toMatchSnapshot();
});