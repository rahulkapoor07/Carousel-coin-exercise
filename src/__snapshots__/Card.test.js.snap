// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should snapshot 1`] = `[Function]`;

exports[`should snapshot 1 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    />
    <img
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
